raven/base.py

```diff
@@ -636,7 +637,10 @@ def capture(self, event_type, data=None, date=None, time_spent=None,
             **kwargs)
 
         # should this event be sampled?
-        if self._random.random() < self.sample_rate:
+        if not sample_rate:
```

Comment:
thats actually probably fine -- if they want to disable sampling they can do sample_rate=1